bitwarden_license/src/Commercial.Core/Billing/ProviderBillingService.cs

```diff
@@ -437,139 +438,136 @@ public async Task<Subscription> SetupSubscription(
         }
     }
 
-    public async Task UpdateSeatMinimums(
-        Provider provider,
-        int enterpriseSeatMinimum,
-        int teamsSeatMinimum)
+    public async Task ChangePlan(ChangeProviderPlanCommand command)
```

Comment:
I just have to check if I can refactor the duplicate call to retrieve provider plans further at this point

Reply:
Sorry not sure what you mean there - I just meant the `ChangeProviderPlanCommand` could just include the `ProviderPlan` object rather than the ID. Everywhere it's being called is just passing in `providerPlan.Id`, so it seems like an unnecessary double-fetch.